refinery/ui/source/js/commons/services/auth-token.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+angular
+  .module('refineryApp')
+  .factory('authTokenService', ['$resource', 'settings',
+    function ($resource, settings) {
+      return $resource(
+        settings.appRoot + settings.refineryApiV2 + '/obtain-auth-token/',
+        {},
+        {
+          query: {
+            method: 'GET',
+            isArray: false
```

Comment:
pretty sure false is the default, https://docs.angularjs.org/api/ngResource/service/$resource

Reply:
I think query is also a default, so I don't think we need to specify unless we have something custom (params)

Reply:
The `method` & `isArray` were using defaults, but I couldn't remove `query` and get things to work.